.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and Push Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+        - master
+
+jobs:
+  build:
+    name: Build NGINX Image
```

Comment:
Ah I think you forgot changing this line 😄

Reply:
Lol I copied and pasted from my own repo.